DOCUMENTATION.md

```diff
@@ -629,17 +701,59 @@ An example request would look like:
     ]);
 ```
 
-</details>
+You can leave out the `older_than` argument to get the 250 most recent messages.
 
+To get older messages, pass the ID of an older notification into the older_than argument. This returns the next 250 oldest messages from the specified notification ID.
 
-#### Response
+### Arguments
 
-If the request is successful, `response` will be an `array`.
+You can leave out any of these arguments to ignore these filters.
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+#### template_type (optional)
+
+You can filter by:
+
+* `email`
+* `sms`
+* `letter`
+
+#### status (optional)
+
+| status | description | text | email | letter |
+|:--- |:--- |:--- |:--- |:--- |
+|`created` |The message is queued to be sent to the provider|Yes|Yes||
+|`sending` |The message is queued to be sent to the recipient by the provider|Yes|Yes||
+|`delivered`|The message was successfully delivered|Yes|Yes||
+|`failed`|This will return all failure statuses:<br>- `permanent-failure`<br>- `temporary-failure`<br>- `technical-failure`|Yes|Yes||
+|`permanent-failure`|The provider was unable to deliver message, email or phone number does not exist; remove this recipient from your list|Yes|Yes||
+|`temporary-failure`|The provider was unable to deliver message, email inbox was full or phone was turned off; you can try to send the message again|Yes|Yes||
+|`technical-failure`|Email or text message: Notify had a technical failure; you can try to send the message again. <br><br>Letter: Notify had an unexpected error while sending to our printing provider. <br><br>You can omit this argument to ignore this filter.|Yes|Yes||
+|`accepted`|Notify is printing and posting the letter|||Yes|
+|`received`|The provider has received the letter to deliver|||Yes|
+
+#### reference (optional)
+
+A unique identifier you can create if necessary. This reference identifies a single unique notification or a batch of notifications.
+
+```php
+reference='STRING' # optional string - identifies notification(s)
```

Comment:
we have this `reference='STRING'` in the docs four times, each way slightly different syntax/comments - we should probs clean it up to make sure they're all the same